### DISCUSSION TITLE
Alarm

### DISCUSSION THREAD

**Discussion by Vince56190** [agronick/aa-torque#26] [Ideas]
Possibility of adding alarms if a value is exceeded. Either a change in the color at the bottom of the dial or a small red light

**Comment by Mtn-Mike** [comment]
I second this idea!

It would be nice if we could set limits in AA Torque that, when achieved or exceeded, the needle (or the ray) or the gauge background would turn a different color (red, yellow...whatever).  

It would be even better if AA Torque could inherit these limits from the Torque app itself:  In Torque you can set "High Warning" and "Low Warning" values.  Don't know if that's possible, but it would be super slick!

**Comment by kacnje** [comment]
Imagine that scenario. AA Torque is enabled in the background. You get an alarm when your car starts DPF regeneration, so you know not to interrupt if you can.

**Comment by agronick** [comment]
Yeah I think this would be a cool idea.
